{"name": "Attempted login with invalid credentials and some empty data -- @1.2 ", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"C:\\Users\\Zahri Hidayat\\.conda\\envs\\Softwar_Testing\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Zahri Hidayat\\.conda\\envs\\Softwar_Testing\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login_steps.py\", line 33, in step_impl\n    assert error_message == real_error_message\n"}, "steps": [{"name": "Given I am on the login page", "status": "passed", "start": 1717323191391, "stop": 1717323200029}, {"name": "When I fill in \"username\" with \"\"", "status": "passed", "start": 1717323200030, "stop": 1717323200030}, {"name": "And I fill in \"Password\" with \"\"", "status": "passed", "start": 1717323200030, "stop": 1717323200031}, {"name": "And I press \"Login\"", "status": "passed", "start": 1717323200031, "stop": 1717323200102}, {"name": "Then I should see the error message \"Epic sadface: Username and password are required\"", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"C:\\Users\\Zahri Hidayat\\.conda\\envs\\Softwar_Testing\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Zahri Hidayat\\.conda\\envs\\Softwar_Testing\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login_steps.py\", line 33, in step_impl\n    assert error_message == real_error_message\n"}, "start": 1717323200102, "stop": 1717323200132}], "parameters": [{"name": "username"}, {"name": "password"}, {"name": "error_message", "value": "Epic sadface: Username and password are required"}], "start": 1717323191390, "stop": 1717323202224, "uuid": "1d7396c3-279c-4b8a-a691-2cd839fc32f2", "historyId": "70c6e089ea0e2a696fe796c24c3a40c7", "fullName": "Login: Attempted login with invalid credentials and some empty data", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "negative"}, {"name": "feature", "value": "Login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}